administrator/language/en-GB/plg_workflow_notification.ini

```
@@ -10,7 +10,7 @@ PLG_WORKFLOW_NOTIFICATION_ADDTEXT="The stage has changed"
 PLG_WORKFLOW_NOTIFICATION_ADDTEXT_DESC="This text will be sent: Title [title], changed by [user], new state: [state]. You can add own text to this information. You can localise the text by using a language key and creating a language override."
 PLG_WORKFLOW_NOTIFICATION_ADDTEXT_LABEL="Additional Message Text"
 PLG_WORKFLOW_NOTIFICATION_ON_TRANSITION_MSG="Title: %1$s. Transition %2$s performed by %3$s. New state: %4$s."
-PLG_WORKFLOW_NOTIFICATION_ON_TRANSITION_SUBJECT="The status of an %1$s has been changed"
+PLG_WORKFLOW_NOTIFICATION_ON_TRANSITION_SUBJECT="The status of \"%1$s\" has been changed"
```

```suggestion
PLG_WORKFLOW_NOTIFICATION_ON_TRANSITION_SUBJECT="The status of \"%s\" has been changed"
```

there is only one variable - yes I know - it was wrong before.